File: Easy/ClimbingStairs/SolutionTest.py
from unittest import TestCase, main
from Solution import Solution

class SolutionTest(TestCase):
    def test_climbStairsTest(self):
        self.assertEquals(2, Solution.climbStairs(2))


    def test_climbStairsTest2(self):
        self.assertEquals(3, Solution.climbStairs(3))


    def test_climbStairsTest3(self):
        self.assertEquals(1134903170, Solution.climbStairs(44))


if __name__ == "__main__":
    main()

File: Easy/ClimbingStairs/SolutionTest.java
package Easy.ClimbingStairs;
import org.junit.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class SolutionTest{

    @Test
    public void climbStairsTest(){
        assertEquals(2, Solution.climbStairs(2));
    }

    @Test
    public void climbStairsTest2(){
        assertEquals(3, Solution.climbStairs(3));
    }

    @Test
    public void climbStairsTest3(){
        assertEquals(1134903170, Solution.climbStairs(44));
    }
}
